### ISSUE TITLE
Is Nuxhash broken for just me or has it stopped working for everyone?

### ISSUE THREAD

**Issue by UbuntuSteve** [YoRyan/nuxhash#51]
Is nicehash blocking nuxhash?

  File "/usr/lib/python3.6/socket.py", line 713, in create_connection
    sock.connect(sa)
ConnectionRefusedError: [Errno 111] Connection refused

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.6/threading.py", line 916, in _bootstrap_inner
    self.run()
  File "/usr/local/lib/python3.6/dist-packages/nuxhash/gui/mining.py", line 342, in run
    self._scheduler.run()
  File "/usr/lib/python3.6/sched.py", line 154, in run
    action(*argument, **kwargs)
  File "/usr/local/lib/python3.6/dist-packages/nuxhash/gui/mining.py", line 405, in _read_status
    for algorithm in running_algorithms}
  File "/usr/local/lib/python3.6/dist-packages/nuxhash/gui/mining.py", line 405, in <dictcomp>
    for algorithm in running_algorithms}
  File "/usr/local/lib/python3.6/dist-packages/nuxhash/miners/miner.py", line 108, in wrapper
    return method(self, *args, **kwargs)
  File "/usr/local/lib/python3.6/dist-packages/nuxhash/miners/excavator.py", line 358, in current_speeds
    raise miner.MinerNotRunning('could not connect to excavator')
nuxhash.miners.miner.MinerNotRunning: could not connect to excavator

**Comment by Michael1095** [comment]
Working fine for me

**Comment by jspanchu** [comment]
Can you try `nuxhashd --show-mining` ?
If it shows something similar to this, then it means your gpu ran out of mem.
`2020-03-06 21:50:54,607 DEBUG: [21:50:54][0x00007f3f7136a700][fatal] wrkr0-0 | CUDA error 'out of memory' in func 'run' line 1442
`

**Comment by jspanchu** [comment]
I think I understand why it is not working for you. I faced the same error that you've hit and came up with a resolution. But I think it depends on your GPU.
My config is
```
OS: Arch Linux
GPU: GTX 1050
CPU: i7-8750H
```
**TLDR;** Your GPU is probably out of RAM and unable to generate DAG file for daggerhashimoto algo. 

I tried to resolve the out of memory error by a hacky solution that disables graphics rendering on the GPU (hence frees up memory). Since it is a laptop with optimus, I was using [optimus-manager](https://github.com/Askannz/optimus-manager) to manage GPU switching. 

So apparently whenever I switched to Nvidia, the desktop composition was done on the GPU (it was accessing the GPU's VRAM about 270-300 MB, this shouldn't be surprising since GNOME is quite a memory hungry desktop environment.) This can be confirmed from the output of `nvidia-smi`
```
 $ nvidia-smi
Sat Mar  7 08:37:19 2020       
+-----------------------------------------------------------------------------+
| NVIDIA-SMI 440.59       Driver Version: 440.59       CUDA Version: 10.2     |
|-------------------------------+----------------------+----------------------+
| GPU  Name        Persistence-M| Bus-Id        Disp.A | Volatile Uncorr. ECC |
| Fan  Temp  Perf  Pwr:Usage/Cap|         Memory-Usage | GPU-Util  Compute M. |
|===============================+======================+======================|
|   0  GeForce GTX 1050    Off  | 00000000:01:00.0 Off |                  N/A |
| N/A   50C    P0    N/A /  N/A |    341MiB /  4042MiB |      3%      Default |
+-------------------------------+----------------------+----------------------+
                                                                               
+-----------------------------------------------------------------------------+
| Processes:                                                       GPU Memory |
|  GPU       PID   Type   Process name                             Usage      |
|=============================================================================|
|    0      1478      G   /usr/lib/Xorg                                 28MiB |
|    0      1666      G   /usr/lib/Xorg                                 62MiB |
|    0      1725      G   /usr/bin/gnome-shell                         179MiB |
+-----------------------------------------------------------------------------+
```

And when nuxhashd tries to mine with the `daggerhashimoto` algo that requires at least 4GB for DAG generation, it fails with the error `2020-03-06 21:50:54,607 DEBUG: [21:50:54][0x00007f3f7136a700][fatal] wrkr0-0 | CUDA error 'out of memory' in func 'run' line 1442`since the GPU runs out of memory. 

 I confirmed that the same happens with [ethminer](https://github.com/ethereum-mining/ethminer) too. Look at the message
`Unexpected error CUDA error in func initEpoch_internal at line 123 out of memory on CUDA device 01:00.0`
```
$ ethminer -P ...


ethminer 0.19.0-alpha.0
Build: linux/release/gnu

No OpenCL platforms found
 i 08:44:22 ethminer Configured pool eu1.ethermine.org:4444
 i 08:44:22 ethminer Selected pool eu1.ethermine.org:4444
 i 08:44:22 ethminer Negotiation of EthereumStratum/2.0.0 failed. Trying another ...
 i 08:44:23 ethminer Negotiation of EthereumStratum/1.0.0 (NiceHash) failed. Trying another ...
 i 08:44:23 ethminer Stratum mode : Eth-Proxy compatible
 i 08:44:23 ethminer Established connection to eu1.ethermine.org [172.65.207.106:4444]
 i 08:44:23 ethminer Spinning up miners...
cu 08:44:23 cuda-0   Using Pci Id : 01:00.0 GeForce GTX 1050 (Compute 6.1) Memory : 3.95 GB
 i 08:44:23 ethminer Epoch : 320 Difficulty : 4.00 Gh
 i 08:44:23 ethminer Job: 2ff29292… block 9622942 eu1.ethermine.org [172.65.207.106:4444]
 i 08:44:24 ethminer Job: ad87a949… block 9622942 eu1.ethermine.org [172.65.207.106:4444]
cu 08:44:25 cuda-0   Generating DAG + Light : 3.55 GB
cu 08:44:25 cuda-0   Unexpected error CUDA error in func initEpoch_internal at line 123 out of memory on CUDA device 01:00.0
cu 08:44:25 cuda-0   Mining suspended ...
^C i 08:44:26 main     Got interrupt ...
 i 08:44:26 main     Disconnected from eu1.ethermine.org [172.65.207.106:4444]
 i 08:44:26 main     Shutting down miners...
 i 08:44:26 main     Terminated!
```

So I switched back to intel with optimus-manager. Then used [acpi_call](https://github.com/mkottman/acpi_call) to turn on my GPU. 
I did not load any nvidia modules since we don't want the GPU rendering our desktop or anything for that matter. So I did the commands:
```
$ optimus-manager --switch intel
```
automatically logs out, so login again
```
$ sudo -s
[sudo] password for user: 
sh-5.0# echo '\_SB.PCI0.PEG0.PEGP._ON' > /proc/acpi/call
sh-5.0# exit
$ nvidia-smi
Sat Mar  7 08:51:25 2020       
+-----------------------------------------------------------------------------+
| NVIDIA-SMI 440.59       Driver Version: 440.59       CUDA Version: 10.2     |
|-------------------------------+----------------------+----------------------+
| GPU  Name        Persistence-M| Bus-Id        Disp.A | Volatile Uncorr. ECC |
| Fan  Temp  Perf  Pwr:Usage/Cap|         Memory-Usage | GPU-Util  Compute M. |
|===============================+======================+======================|
|   0  GeForce GTX 1050    Off  | 00000000:01:00.0 Off |                  N/A |
| N/A   49C    P0    N/A /  N/A |      0MiB /  4042MiB |      1%      Default |
+-------------------------------+----------------------+----------------------+
                                                                               
+-----------------------------------------------------------------------------+
| Processes:                                                       GPU Memory |
|  GPU       PID   Type   Process name                             Usage      |
|=============================================================================|
|  No running processes found                                                 |
+-----------------------------------------------------------------------------+

```

See the `No running processes found`? Also note that now, Memory-Usage is`0MiB /  4042MiB`. That is a positive sign.
now `nuxhashd --show-mining` works. Same goes for ethminer too.
```
$ nuxhashd --show-mining
2020-03-07 08:56:04,450 INFO: Querying NiceHash for miner connection information...
2020-03-07 08:56:04,454 DEBUG: Starting new HTTPS connection (1): api2.nicehash.com:443
2020-03-07 08:56:04,731 DEBUG: https://api2.nicehash.com:443 "GET /main/api/v2/public/simplemultialgo/info/ HTTP/1.1" 200 None
2020-03-07 08:56:04,736 DEBUG: Starting new HTTPS connection (1): api2.nicehash.com:443
2020-03-07 08:56:04,829 DEBUG: https://api2.nicehash.com:443 "GET /main/api/v2/mining/algorithms/ HTTP/1.1" 200 None
2020-03-07 08:56:04,847 DEBUG: =========================== www.nicehash.com =========================
2020-03-07 08:56:04,847 DEBUG:            Excavator v1.5.14a GPU Miner for NiceHash.
2020-03-07 08:56:04,847 DEBUG:            Copyright (C) 2018 NiceHash. All rights reserved.
2020-03-07 08:56:04,847 DEBUG: =========================== www.nicehash.com =========================
2020-03-07 08:56:04,847 DEBUG: Build time: 2018-12-07 10:21:06+01:00
2020-03-07 08:56:04,847 DEBUG: Build number: 3897564716
2020-03-07 08:56:04,847 DEBUG: [08:56:04][0x00007f18e149ac80][info] Log started
2020-03-07 08:56:05,038 DEBUG: [08:56:05][0x00007f18e149ac80][info] core | Found CUDA device: GeForce GTX 1050
2020-03-07 08:56:05,173 DEBUG: [08:56:05][0x00007f18e149ac80][info] api | Listening on 127.0.0.1:47613
2020-03-07 08:56:05,173 DEBUG: [08:56:05][0x00007f18e149ac80][info] core | Initialized!
2020-03-07 08:56:05,175 DEBUG: Starting new HTTPS connection (1): api2.nicehash.com:443
2020-03-07 08:56:05,265 DEBUG: [08:56:05][0x00007f18e149ac80][info] Connected to nhmp.eu.nicehash.com:3200
2020-03-07 08:56:05,289 DEBUG: [08:56:05][0x00007f18d5fff700][info] nhmp | Subscribed
2020-03-07 08:56:05,448 DEBUG: https://api2.nicehash.com:443 "GET /main/api/v2/public/simplemultialgo/info/ HTTP/1.1" 200 None
2020-03-07 08:56:05,453 INFO: Assigning nvidia_GPU-68943c48-ab2a-f1f0-7ca1-21cd65fd3299 to excavator_daggerhashimoto (0.0 mBTC/day)
2020-03-07 08:56:05,482 DEBUG: [08:56:05][0x00007f18e149ac80][info] nhmp | daggerhashimoto | add success
2020-03-07 08:56:05,530 DEBUG: [08:56:05][0x00007f18d5fff700][info] net | daggerhashimoto | New job '00000000e16274a3', diff=2
2020-03-07 08:56:05,713 DEBUG: [08:56:05][0x00007f18d136a700][info] wrkr0-0 | Algorithm: CUDA-daggerhashimoto parameters: B=8192,TPB=128,HPW=4,S=1
2020-03-07 08:56:05,713 DEBUG: [08:56:05][0x00007f18d136a700][info] wrkr0-0 | Mining with high intensity
2020-03-07 08:56:05,715 DEBUG: [08:56:05][0x00007f18d136a700][info] job-daggerhashimoto | Generating DAG cache (5230805f10adff2c)...
2020-03-07 08:56:07,392 DEBUG: [08:56:07][0x00007f18e149ac80][info] net | daggerhashimoto | New job '00000000e16328be', diff=2
2020-03-07 08:56:09,511 DEBUG: [08:56:09][0x00007f18d136a700][info] job-daggerhashimoto | DAG cache (5230805f10adff2c) generation completed (3794ms)
2020-03-07 08:56:14,990 DEBUG: [08:56:14][0x00007f18d5fff700][info] net | daggerhashimoto | New job '00000000e16451ec', diff=2
2020-03-07 08:56:21,424 DEBUG: [08:56:21][0x00007f18e149ac80][info] net | daggerhashimoto | New job '00000000e1650721', diff=2
2020-03-07 08:56:28,490 DEBUG: [08:56:28][0x00007f18d5fff700][info] net | daggerhashimoto | New job '00000000e165df52', diff=2
2020-03-07 08:56:35,560 DEBUG: [08:56:35][0x00007f18e149ac80][info] net | daggerhashimoto | New job '00000000e166de95', diff=2
2020-03-07 08:56:42,625 DEBUG: [08:56:42][0x00007f18d5fff700][info] net | daggerhashimoto | New job '00000000e1678c5a', diff=2
2020-03-07 08:56:45,733 DEBUG: [08:56:45][0x00007f18e149ac80][info] net | daggerhashimoto | New job '00000000e16814fb', diff=2
2020-03-07 08:56:46,151 DEBUG: [08:56:46][0x00007f18d5fff700][info] net | daggerhashimoto | New job '00000000e1688957', diff=2

```

```
$ ethminer -P ...


ethminer 0.19.0-alpha.0
Build: linux/release/gnu

No OpenCL platforms found
 i 08:57:14 ethminer Configured pool eu1.ethermine.org:4444
 i 08:57:14 ethminer Selected pool eu1.ethermine.org:4444
 i 08:57:15 ethminer Negotiation of EthereumStratum/2.0.0 failed. Trying another ...
 i 08:57:15 ethminer Negotiation of EthereumStratum/1.0.0 (NiceHash) failed. Trying another ...
 i 08:57:15 ethminer Stratum mode : Eth-Proxy compatible
 i 08:57:15 ethminer Established connection to eu1.ethermine.org [172.65.207.106:4444]
 i 08:57:15 ethminer Spinning up miners...
cu 08:57:15 cuda-0   Using Pci Id : 01:00.0 GeForce GTX 1050 (Compute 6.1) Memory : 3.95 GB
 i 08:57:15 ethminer Epoch : 320 Difficulty : 4.00 Gh
 i 08:57:15 ethminer Job: 8df47ded… block 9622999 eu1.ethermine.org [172.65.207.106:4444]
 i 08:57:17 ethminer Job: f9cd99bc… block 9622999 eu1.ethermine.org [172.65.207.106:4444]
cu 08:57:17 cuda-0   Generating DAG + Light : 3.55 GB
...
...
...
 i 08:57:18 ethminer Job: b29845a3… block 9623000 eu1.ethermine.org [172.65.207.106:4444]
 m 08:58:59 ethminer 0:01 A2 12.40 Mh - cu0 12.40
 i 08:59:03 ethminer Job: d2812c38… block 9623005 eu1.ethermine.org [172.65.207.106:4444]
 m 08:59:04 ethminer 0:01 A2 12.39 Mh - cu0 12.39
 i 08:59:07 ethminer Job: 26661cb3… block 9623005 eu1.ethermine.org [172.65.207.106:4444]
```

Note to the developer: Is there a way to tell nuxhash not to run specific algorithms? I tried unchecking the daggerhashimoto variants, re-run benchmarks with daggerhashimoto options unchecked and started mining, yet it uses daggerhashimoto.

EDIT: Grammar and formatting

**Comment by YoRyan** [comment]
Impressive troubleshooting!

No, there is currently no formal way to disable algorithms. The checkmarks only select which ones to benchmark. As a workaround, you could set their hash rates to 0 to keep the profit switcher from selecting them.

**Comment by sku10** [comment]
I tried nuxhash. Gui version doesn't work, problem with wx. nuxhashd starts.  seemingly connects to nicehash but nicehash reports accepted speed 0.00000 MH/s, rejected speed 0.00000 MH/s. so it seems it is receiving no communication from nuxhashd  even thou the local screenoutput shows activity and GPU fans speed up when running nushashd.  


```
$ nuxhashd --show-mining

2021-05-17 07:07:54,193 INFO: Querying NiceHash for miner connection information...
2021-05-17 07:07:54,195 DEBUG: Starting new HTTPS connection (1): api2.nicehash.com:443
2021-05-17 07:07:54,383 DEBUG: https://api2.nicehash.com:443 "GET /main/api/v2/public/simplemultialgo/info/ HTTP/1.1" 200 None
2021-05-17 07:07:54,386 DEBUG: Starting new HTTPS connection (1): api2.nicehash.com:443
2021-05-17 07:07:54,491 DEBUG: https://api2.nicehash.com:443 "GET /main/api/v2/mining/algorithms/ HTTP/1.1" 200 None
2021-05-17 07:07:54,501 DEBUG: =========================== www.nicehash.com =========================
2021-05-17 07:07:54,501 DEBUG:            Excavator v1.5.14a GPU Miner for NiceHash.
2021-05-17 07:07:54,501 DEBUG:            Copyright (C) 2018 NiceHash. All rights reserved.
2021-05-17 07:07:54,501 DEBUG: =========================== www.nicehash.com =========================
2021-05-17 07:07:54,501 DEBUG: Build time: 2018-12-07 10:21:06+01:00
2021-05-17 07:07:54,501 DEBUG: Build number: 3897564716
2021-05-17 07:07:54,501 DEBUG: [07:07:54][0x00007f28ee3b6680][info] Log started
2021-05-17 07:07:54,512 DEBUG: [07:07:54][0x00007f28ee3b6680][info] core | Found CUDA device: NVIDIA GeForce RTX 3060
2021-05-17 07:07:54,559 DEBUG: [07:07:54][0x00007f28ee3b6680][info] api | Listening on 127.0.0.1:53065
2021-05-17 07:07:54,559 DEBUG: [07:07:54][0x00007f28ee3b6680][info] core | Initialized!
2021-05-17 07:07:54,560 DEBUG: Starting new HTTPS connection (1): api2.nicehash.com:443
2021-05-17 07:07:54,662 DEBUG: [07:07:54][0x00007f28ee3b6680][info] Connected to nhmp.eu.nicehash.com:3200
2021-05-17 07:07:54,715 DEBUG: [07:07:54][0x00007f28ed2c4700][info] nhmp | Subscribed
2021-05-17 07:07:54,744 DEBUG: https://api2.nicehash.com:443 "GET /main/api/v2/public/simplemultialgo/info/ HTTP/1.1" 200 None
2021-05-17 07:07:54,745 INFO: Assigning nvidia_GPU-251a4812-a3ca-7c8f-f25a-9d14a6d03f4e to excavator_neoscrypt (0.0 mBTC/day)
2021-05-17 07:07:54,797 DEBUG: [07:07:54][0x00007f28ee3b6680][info] nhmp | neoscrypt | add success
2021-05-17 07:07:54,804 DEBUG: [07:07:54][0x00007f28e0cf2700][info] wrkr0-0 | Algorithm: CUDA-neoscrypt parameters: B=102 M=1
2021-05-17 07:07:54,804 DEBUG: [07:07:54][0x00007f28e0cf2700][info] wrkr0-0 | Mining with high intensity
2021-05-17 07:07:54,833 DEBUG: [07:07:54][0x00007f28ed2c4700][info] net | neoscrypt | New job '00000000f0fa2594', diff=0.0233509
2021-05-17 07:08:07,077 DEBUG: [07:08:07][0x00007f28ee3b6680][info] net | neoscrypt | New job '00000000f0fa2595', diff=0.0233509
2021-05-17 07:08:51,367 DEBUG: [07:08:51][0x00007f28ed2c4700][info] net | neoscrypt | New job '00000000f0fa2596', diff=0.0233509
2021-05-17 07:08:54,751 DEBUG: Starting new HTTPS connection (1): api2.nicehash.com:443
2021-05-17 07:08:54,981 DEBUG: https://api2.nicehash.com:443 "GET /main/api/v2/public/simplemultialgo/info/ HTTP/1.1" 200 None
2021-05-17 07:09:37,707 DEBUG: [07:09:37][0x00007f28ee3b6680][info] net | neoscrypt | New job '00000000f0fa2597', diff=0.0233509
2021-05-17 07:09:54,983 DEBUG: Starting new HTTPS connection (1): api2.nicehash.com:443
2021-05-17 07:09:55,151 DEBUG: https://api2.nicehash.com:443 "GET /main/api/v2/public/simplemultialgo/info/ HTTP/1.1" 200 None
2021-05-17 07:10:13,050 DEBUG: [07:10:13][0x00007f28ed2c4700][info] net | neoscrypt | New job '00000000f0fa2598', diff=0.015625
2021-05-17 07:10:14,138 DEBUG: [07:10:14][0x00007f28ee3b6680][info] net | neoscrypt | New job '00000000f0fa2599', diff=0.015625
2021-05-17 07:10:24,136 DEBUG: [07:10:24][0x00007f28ed2c4700][info] net | neoscrypt | New job '00000000f0fa259a', diff=0.015625
2021-05-17 07:10:34,137 DEBUG: [07:10:34][0x00007f28ee3b6680][info] net | neoscrypt | New job '00000000f0fa259b', diff=0.015625
2021-05-17 07:10:44,140 DEBUG: [07:10:44][0x00007f28ed2c4700][info] net | neoscrypt | New job '00000000f0fa259c', diff=0.015625
2021-05-17 07:10:54,143 DEBUG: [07:10:54][0x00007f28ee3b6680][info] net | neoscrypt | New job '00000000f0fa259d', diff=0.015625
2021-05-17 07:10:55,153 DEBUG: Starting new HTTPS connection (1): api2.nicehash.com:443
2021-05-17 07:10:55,330 DEBUG: https://api2.nicehash.com:443 "GET /main/api/v2/public/simplemultialgo/info/ HTTP/1.1" 200 None
2021-05-17 07:11:04,141 DEBUG: [07:11:04][0x00007f28ed2c4700][info] net | neoscrypt | New job '00000000f0fa259e', diff=0.015625
2021-05-17 07:11:14,142 DEBUG: [07:11:14][0x00007f28ee3b6680][info] net | neoscrypt | New job '00000000f0fa259f', diff=0.015625
2021-05-17 07:11:24,159 DEBUG: [07:11:24][0x00007f28ed2c4700][info] net | neoscrypt | New job '00000000f0fa25a0', diff=0.015625
2021-05-17 07:11:34,144 DEBUG: [07:11:34][0x00007f28ee3b6680][info] net | neoscrypt | New job '00000000f0fa25a1', diff=0.015625
2021-05-17 07:11:44,151 DEBUG: [07:11:44][0x00007f28ed2c4700][info] net | neoscrypt | New job '00000000f0fa25a2', diff=0.015625
2021-05-17 07:11:54,142 DEBUG: [07:11:54][0x00007f28ee3b6680][info] net | neoscrypt | New job '00000000f0fa25a3', diff=0.015625
2021-05-17 07:11:55,332 DEBUG: Starting new HTTPS connection (1): api2.nicehash.com:443
2021-05-17 07:11:55,507 DEBUG: https://api2.nicehash.com:443 "GET /main/api/v2/public/simplemultialgo/info/ HTTP/1.1" 200 None
2021-05-17 07:12:04,146 DEBUG: [07:12:04][0x00007f28ed2c4700][info] net | neoscrypt | New job '00000000f0fa25a4', diff=0.015625
2021-05-17 07:12:12,058 DEBUG: [07:12:12][0x00007f28ee3b6680][info] net | neoscrypt | New job '00000000f0fa25a5', diff=0.015625
2021-05-17 07:12:33,707 DEBUG: [07:12:33][0x00007f28ed2c4700][info] net | neoscrypt | New job '00000000f0fa25a6', diff=0.015625
2021-05-17 07:12:55,509 DEBUG: Starting new HTTPS connection (1): api2.nicehash.com:443
2021-05-17 07:12:55,698 DEBUG: https://api2.nicehash.com:443 "GET /main/api/v2/public/simplemultialgo/info/ HTTP/1.1" 200 None
2021-05-17 07:13:29,447 DEBUG: [07:13:29][0x00007f28ee3b6680][info] net | neoscrypt | New job '00000000f0fa25a7', diff=0.015625
```

![Screenshot from 2021-05-17 07-38-58](https://user-images.githubusercontent.com/37125254/118433124-fb4e2a00-b6e2-11eb-8607-f395b936ed48.png)

**Comment by CocolinoFan** [comment]
**CUDA error 'out of memory' in func 'run' line 1442**

```
2022-10-30 19:00:57,028 DEBUG: [19:00:57][0x00007fefcee886c0][info] net | daggerhashimoto | New job '00000000307388d2', diff=0.378959
2022-10-30 19:00:59,164 DEBUG: [19:00:59][0x00007fefcee886c0][info] net | daggerhashimoto | New job '00000000307388d3', diff=0.378959
2022-10-30 19:01:03,621 DEBUG: [19:01:03][0x00007fefcee886c0][info] net | daggerhashimoto | New job '00000000307388d4', diff=0.378959
2022-10-30 19:01:06,530 DEBUG: [19:01:06][0x00007fefc1fff6c0][info] job-daggerhashimoto | Generating DAG cache (27027a96b05f67c6)...
2022-10-30 19:01:09,681 DEBUG: [19:01:09][0x00007fefcee886c0][info] net | daggerhashimoto | New job '00000000307388d5', diff=0.378959
2022-10-30 19:01:12,714 DEBUG: [19:01:12][0x00007fefcee886c0][info] net | daggerhashimoto | New job '00000000307388d6', diff=0.378959
2022-10-30 19:01:13,031 DEBUG: [19:01:13][0x00007fefcee886c0][info] net | daggerhashimoto | New job '00000000307388d7', diff=0.378959
2022-10-30 19:01:13,640 DEBUG: [19:01:13][0x00007fefc1fff6c0][info] job-daggerhashimoto | DAG cache (27027a96b05f67c6) generation completed (7049ms)
2022-10-30 19:01:13,755 DEBUG: [19:01:13][0x00007fefc1fff6c0][fatal] wrkr0-1 | CUDA error 'out of memory' in func 'run' line 1442
2022-10-30 19:01:13,756 DEBUG: [19:01:13][0x00007fefd2b75740][info] Shutting down
2022-10-30 19:01:13,756 DEBUG: [19:01:13][0x00007fefd2b75740][info] api | Closing
```

EDIT: Formatting

**Comment by UbuntuSteve** [comment]
Since mining is dead so is this.

Love you bye.
